docs/source/package_orderers.rst

```diff
@@ -0,0 +1,86 @@
+================
+Package Orderers
+================
+
+Rez's default version resolution algorithm will always sort by the latest alphanumeric
```

Comment:
Should we reference https://rez.readthedocs.io/en/stable/basic_concepts.html#versions? I would also like to keep some parts of https://rez--1737.org.readthedocs.build/en/1737/configuring_rez.html#package_orderers if possible.

Reply:
@JeanChristopheMorinPerso I added the cross reference to https://rez.readthedocs.io/en/stable/basic_concepts.html#versions.

What do you mean you'd like to keep some parts of https://rez--1737.org.readthedocs.build/en/1737/configuring_rez.html#package_orderers. Can you be more specific?